biz.aQute.bnd.gradle/test/aQute/bnd/gradle/TestBndPlugin.groovy

```diff
@@ -123,7 +123,7 @@ class TestBndPlugin extends Specification {
           bndrun.isFile()
           props.load(bndrun, new Slf4jReporter(TestBndPlugin.class))
           props.getProperty('-runbundles') =~ /osgi\.enroute\.junit\.wrapper/
-          result.output =~ Pattern.quote('Unresolved requirements: [test.simple]')
+          result.output =~ '(?s)Unresolved requirements:(.*)\\s+test.simple\\n'
```

Comment:
This fails on Windows: https://ci.appveyor.com/project/bndtools/bnd/build/2078/job/l27nqjlq7vg3on5b#L313